sandbox-tools/merge-repos/src/config.ts

```diff
@@ -69,12 +69,14 @@ export const reposToSyncAndMerge: IRepoSyncDetails = {
         url: "https://github.com/open-telemetry/opentelemetry-js",
         branch: "main",
         //mergeStartPoint: "HEAD",    // Used for local testing to validate periodic execution
+        //branchStartPoint: "e91cac503c0bab95b429ff3c4b23249653882054",
```

Comment:
This was used to both test the "upgrading" as well as run the script to "fix" the previous merges that resulted in missing files, folders and incorrect content of some files (ie. `auto-merge/contrib` versions of files like `tsconfig.json` and `index.ts` being merged into (or removed from) the `auto-merge/js`